spec/integration/outputs/delete_spec.rb

```diff
@@ -0,0 +1,62 @@
+require_relative "../../../spec/es_spec_helper"
+require "logstash/outputs/elasticsearch"
+
+
+describe "Versioned delete", :integration => true, :version_greater_than_equal_to_2x => true do
+  require "logstash/outputs/elasticsearch"
+
+  before :each do
+    @es = get_client
```

Comment:
Can you make the changes I've requested in `index_version_spec` apply here as well?